.github/workflows/go.yml

```diff
@@ -0,0 +1,131 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build-and-test:
+    name: Build Go and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        
+        # --prune --unshallow is needed for changelog generation by goreleaser
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        run: |
+          git fetch --prune --unshallow
+        
+      - name: Unshallow
```

Comment:
this step is not needed any more.

Reply:
Thanks, fixed!

Reply:
Okay, so it turns out the examples from https://github.com/actions/checkout#Fetch-all-history-for-all-tags-and-branches seem to be wrong or outdated. I got the following message when trying to do use 

```
      - name: Check out code into the Go module directory
        uses: actions/checkout@v2
        # needed for changelog generation by goreleaser
        run: |
          git fetch --prune --unshallow 
```

> a step cannot have both the uses and run keys